text/000x-assethub.md

```diff
@@ -0,0 +1,92 @@
+# RFC-0000: Lowering NFT Deposits on Polkadot and Kusama Asset Hubs
+
+|                 |                                                                                                    |
+| --------------- | -------------------------------------------------------------------------------------------------- |
+| **Start Date**  | 2 November 2023                                                                                    |
+| **Description** | A proposal to reduce the minimum deposit required for collection creation on the Polkadot and Kusama Asset Hub, making it more accessible and affordable for artists. |
+| **Authors**     | [Aurora Poppyseed](https://github.com/poppyseedDev), [Just_Luuuu](https://github.com/justLuuuu), [VikiiVal](https://github.com/vikiival)  |
+
+## Summary
+
+This RFC proposes a change to the current deposit requirements on the Polkadot and Kusama Asset Hub for creating NFT collections. The objective is to lower the barrier to entry for artists, fostering a more inclusive and vibrant ecosystem, while maintaining network integrity and preventing spam.
+
+## Motivation
+
+The current deposit of 10 DOT for collection creation on the Polkadot Asset Hub presents a significant financial barrier for many artists. By lowering the deposit requirements, we aim to encourage more artists to participate in the Polkadot NFT ecosystem, thereby enriching the diversity and vibrancy of the community and its offerings.
+
+## Stakeholders
+
+- **Artists**: Primary beneficiaries of the proposed change, particularly those who found the current deposit requirements prohibitive.
+- **KodaDot**: As the facilitator of artists' relations, KodaDot has a vested interest in making the platform more accessible.
```

Comment:
KodaDot may be an example, but I don't think RFCs should target specific entities.
```suggestion
- **NFT Platforms**: As the facilitator of artists' relations, KodaDot has a vested interest in making the platform more accessible.
```